### PR TITLE
Fix compilation of benchmark for parquet writer.

### PR DESCRIPTION
**Pull request by bdice**
This fixes a compilation error introduced in #9810. Tagging @devavret @vuule for review. Feel free to push to this PR with any fixes.

### CODE REVIEW

File: cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp [hunk 1]
```diff
@@ -85,7 +86,7 @@ void BM_parq_write_varying_options(benchmark::State& state)
       cudf_io::parquet_writer_options::builder(source_sink.make_sink_info(), view)
         .compression(compression)
         .stats_level(enable_stats)
-        .column_chunks_file_path(file_path);
+        .column_chunks_file_paths(file_paths);
```
Comment (vuule) [inline cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp:89]: How about this? No need for additional variable.
```suggestion
        .column_chunks_file_paths({file_path});
```
?

File: cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp [hunk 2]
```diff
@@ -68,6 +68,7 @@ void BM_parq_write_varying_options(benchmark::State& state)
   auto const compression  = static_cast<cudf::io::compression_type>(state.range(0));
   auto const enable_stats = static_cast<cudf::io::statistics_freq>(state.range(1));
   auto const file_path    = state.range(2) != 0 ? "unused_path.parquet" : "";
+  auto const file_paths   = std::vector<std::string>{file_path};
```
Comment (bdice) [inline cpp/benchmarks/io/parquet/parquet_writer_benchmark.cpp:71]:
```suggestion
```

**Review by vuule — APPROVED** [review]

**Review by ttnghia — APPROVED** [review]

**Comment by bdice** [comment]
Java CI is failing, seemingly because of another change from #9810. https://gpuci.gpuopenanalytics.com/job/rapidsai/job/gpuci/job/cudf/job/prb/job/cudf-java/CUDA=11.5,GPU_LABEL=gpu,LINUX_VER=centos7,PYTHON=3.7/3289/console

```
20:18:44      [exec] /workspace/java/src/main/native/src/TableJni.cpp: In function ‘void cudf::jni::{anonymous}::createTableMetaData(JNIEnv*, jint, _jobjectArray*&, _jintArray*&, _jbooleanArray*&, _jobjectArray*&, _jobjectArray*&, _jbooleanArray*&, _jintArray*&, _jbooleanArray*&, cudf::io::table_input_metadata&)’:
20:18:49      [exec] /workspace/java/src/main/native/src/TableJni.cpp:746:14: error: ‘class cudf::io::table_input_metadata’ has no member named ‘user_data’
20:18:49      [exec]   746 |     metadata.user_data[meta_keys[i].get()] = meta_values[i].get();
20:18:49      [exec]       |              ^~~~~~~~~
20:18:49      [exec] make[2]: *** [CMakeFiles/cudfjni.dir/build.make:286: CMakeFiles/cudfjni.dir/src/TableJni.cpp.o] Error 1
```

Points to this line: https://github.com/rapidsai/cudf/blob/fc2a32a1576d97a48b0c1c983ef4b31285267e96/java/src/main/native/src/TableJni.cpp#L746

...but the `user_data` field was [removed](https://github.com/rapidsai/cudf/pull/9810/files#diff-64d029b79f74c57bdb9b7671f16248a8807932b9559a916dcad0ea3b789d585bL377) in #9810.

**Comment by bdice** [comment]
I think the Java CI fix is out of scope for this PR. I looked at it but it seems nontrivial and more deeply linked to the changes in #9810. Since #9810 was merged with this Java failure, I will merge this PR to fix the benchmark compilation error. The Java failure can be separately fixed.

**Comment by bdice** [comment]
@gpucibot merge

**Pull request merged**